src/aave-v2/EntryPositionsManager.sol

```diff
@@ -194,12 +190,8 @@ contract EntryPositionsManager is IEntryPositionsManager, PositionsManagerUtils
         if (marketPauseStatus[_poolToken].isBorrowPaused) revert BorrowIsPaused();
 
         ERC20 underlyingToken = ERC20(market.underlyingToken);
-        DataTypes.ReserveConfigurationMap memory reserveConfig = pool.getConfiguration(
-            address(underlyingToken)
-        );
-
-        if (!reserveConfig.getBorrowingEnabled()) revert BorrowingNotEnabled();
-        if (reserveConfig.getFrozen()) revert FrozenOnPool();
+        if (!pool.getConfiguration(address(underlyingToken)).getBorrowingEnabled())
```

Comment:
Should we keep this one after our discussion?

Reply:
It seems we shouldn't?

Reply:
That's what I understand on my side

Reply:
Oh but no we need it because the asset could not be borrowable on Aave

Reply:
Yes I think I'm a bit confused because that check existed before upgrade-morpho-1

Reply:
Ok let's close the PR then